### ISSUE TITLE
Atcoder live contest .

### ISSUE THREAD

**Issue by krsinghshubham** [pustaczek/icie#42]
While pasting the live link for Atcoder the error message said " this event hasnt started yet " or somethig.
Things i did: 
Tried to parse many times during the test, although worked fine after the contest got over.
Tries to paste the link both in alt+F9 and alt+F12 window and also for a particular problem.

Any fixes to avoid this for future contests ? Something to do with tiime- zone ?

**Comment by pustaczek** [comment]
No, the actual time is not relevant, as this is only detected based on AtCoder responding with [a 404 and a "Permission denied." string at https://atcoder.jp/contests/{}/tasks](https://github.com/pustaczek/icie/blob/0039f8a84b971f9a07d38d84b5d34444d65735e9/unijudge-atcoder/src/lib.rs#L291-L303). I don't think being logged in should matter (were you logged in before the contest?), unless they changed something. I will try to check whether something has broken the next time I can attend a contest.

**Comment by krsinghshubham** [comment]
yeah i am pretty sure i was logged in as in parallel i was submitting the solutions  manually.

will wait for the next contest to give more inputs regarding this. thanks.

**Comment by pustaczek** [comment]
I mean logged in inside the extension. Whether you're logged in a browser doesn't matter, the extension would work even without one installed.

**Comment by krsinghshubham** [comment]
So I checked it today and faced the same issue. 
Here is what I did: 
After the contest got finished I tried opening the same contest and it worked. 
Then I logged in and submitted.  (Means now I am logged in the Extension, When i was trying to login before hand using : ICIE passwords it wasn't working)
But now when I am trying to get into new running contest it gives same error as "even not started".
Long contest running now: https://atcoder.jp/contests/hokudai-hitachi2020
